x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/hooks/use_metrics_charts.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import useAsync from 'react-use/lib/useAsync';
+import type { LensBreakdownConfig } from '@kbn/lens-embeddable-utils/config_builder';
+import { findInventoryModel } from '@kbn/metrics-data-access-plugin/common';
+
+export const useMetricsCharts = ({ dataViewId }: { dataViewId?: string }) => {
+  const model = findInventoryModel('host');
+
+  const { value: charts = [] } = useAsync(async () => {
+    const { cpu, disk, memory, network } = await model.metrics.getCharts();
+
+    return [
+      cpu.xy.cpuUsage,
+      cpu.xy.normalizedLoad1m,
+      memory.xy.memoryUsage,
+      memory.xy.memoryFree,
+      disk.xy.diskUsage,
+      disk.xy.diskSpaceAvailable,
+      disk.xy.diskIORead,
+      disk.xy.diskIOWrite,
+      disk.xy.diskReadThroughput,
+      disk.xy.diskWriteThroughput,
+      network.xy.rx,
+      network.xy.tx,
+    ].map((chart) => ({
+      ...chart,
+      layers: chart.layers.map((layer) =>
+        layer.type === 'series'
+          ? {
+              ...layer,
+              breakdown: {
```

Comment:
Nit: This can be a constant of `LensBreakdownConfig` type it's also ok to leave it like that

Reply:
Thanks for the suggestion! Since this object is only used here, I'll leave this as it is.

Reply:
Yes, true we don't use it in the other charts - we can leave it